### ISSUE TITLE
Dependency pattern matching fails

### ISSUE THREAD

**Issue by erbridge** [aurapm/aura#306]
```
$ aura -A ardour-git 
aura >>= Determining dependencies...
aura >>= Dependency checking failed for these reasons:
The dependency `lilv` demands version `<=1.0.0`, but the most recent version is `0.20.0-1`.
```

**Comment by fosskers** [comment]
Thanks for reporting this. Dep parse fails like this won't be a problem in Aura2, as it will use PKGINFO instead of parsing PKGBUILDs manually.

**Comment by erbridge** [comment]
Any rough estimate of how close Aura2 is to release?

**Comment by fosskers** [comment]
Pretty far still, unfortunately. Progress should be made in the coming months when I have more time.

**Comment by erbridge** [comment]
Alright, I'll work around it for now. Thanks for the update.

**Comment by fosskers** [comment]
No problem.

**Comment by fosskers** [comment]
This should no longer be a problem on `master`, which will be released soon as Aura 2.0.
